usaspending_api/database_scripts/matview_generator/summary_state_view.json

```diff
@@ -27,19 +27,18 @@
     "LEFT OUTER JOIN",
     "  transaction_fpds ON (transaction_normalized.id = transaction_fpds.transaction_id)",
     "LEFT OUTER JOIN",
-    "  references_location AS place_of_performance ON (transaction_normalized.place_of_performance_id = place_of_performance.location_id)",
+    "  transaction_fabs ON (transaction_normalized.id = transaction_fabs.transaction_id)",
     "WHERE",
     "  transaction_normalized.action_date >= '2007-10-01' AND",
-    "  place_of_performance.location_country_code = 'USA' AND",
-    "  place_of_performance.state_code IS NOT NULL",
+    "  COALESCE(transaction_fpds.place_of_perform_country_c, transaction_fabs.place_of_perform_country_c, 'USA') = 'USA' AND",
```

Comment:
Previous logic did not default to USA

Reply:
I believe the previous logic was faulty.  Pretty much everywhere else missing country codes are treated as USA.  We had a ticket many moons ago that stated that as a requirement, but this particular matview was not listed.

https://federal-spending-transparency.atlassian.net/browse/DEV-2911

Reply:
Alright, that checks out. I'd still advise a technical note at the top like for the legal_entity PR